app/src/main/java/com/example/simonhsieh/ddcharactersheet/model/CharacterDatabaseManager.java

```diff
@@ -0,0 +1,53 @@
+package com.example.simonhsieh.ddcharactersheet.model;
+
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.content.Context;
+
+/**
+ * Created by simonhsieh on 3/9/18.
+ */
+public final class CharacterDatabaseManager extends SQLiteOpenHelper{
+    private static final String DATABASE_NAME = "characterSheets";
+    private static final int DATABASE_VERSION = 1;
+    private static final String TABLE_NAME = "characters";
+    private static final String COLUMN_NAME = "name";
+    private static final String COLUMN_RACE_ID = "race_id";
+    private static final String RACE_TABLE_NAME = "races";
+//    private static final String COLUMN_STATS_ID = "stats_id";
+//    private static final String STATS_TABLE_NAME = "stats";
+
+    public CharacterDatabaseManager(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+        // TODO: Figure out how I want to store spells for a character.. join table with a generic spells list probably?
+        String CREATE_CONTACTS_TABLE = "CREATE TABLE" + TABLE_NAME + "("
```

Comment:
The table's called contacts?

nit: All caps with underscores is reserved for private static final variables in Java. You can move this to the top of the file. I also enjoy my messy, thatched together constants not cluttering up my core business logic.